src/less/dropdowns.less

```diff
@@ -350,6 +350,47 @@
 
 .dropdown__menu__container {
   padding: @dropdown-padding;
+
+  .resource__wrapper {
+    @media (max-width: @global-screen-small-max) {
+      padding: @dropdown-padding-vertical;
+    }
+
+    .resource__intro__media {
+      @media (max-width: @global-screen-small-max) {
+        width: 3rem;
```

Comment:
We have a global variable `base-h1-font-size` for 3rem. Use that variable instead of 3rem. Applicable for everywhere.